src/components/tricks/PostTrick.js

```diff
@@ -193,7 +202,16 @@ const PostTrick = () => {
               required
               value={difficultyLevel}
               placeholder="8"
-              onChange={(e) => setDifficultyLevel(parseInt(e.target.value))}
+              onChange={(e) => {
+                setDifficultyLevel(Math.max(0, e.target.value));
+              }}
+              onBlur={(e) => {
+                const value = parseInt(e.target.value);
```

Comment:
Absolute minor nitpick, so feel free to ignore:
I really like the change in the other file from calling the variable "value" to calling it "year". This would be applicable here, too.

Reply:
Good point! I will change this later

Reply:
Might even become irrelevant due to the previous comment

Reply:
On second thought: I think this is obsolete, since I changed it to  
`onChange={(e) => { setDifficultyLevel(Math.max(0, e.target.value)); }`